src/Features/Core/Portable/ConflictMarkerResolution/AbstractConflictMarkerCodeFixProvider.cs

```diff
@@ -78,27 +79,40 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
                 return false;
             }
 
-            if (!TryGetConflictLines(text, position, out startLine, out middleLine, out endLine))
+            if (!TryGetConflictLines(text, position, out startLine, out firstMiddleLine, out secondMiddleLine, out endLine))
                 return false;
 
             var startTrivia = root.FindTrivia(startLine.Start);
-            var middleTrivia = root.FindTrivia(middleLine.Start);
+            var firstMiddleTrivia = root.FindTrivia(firstMiddleLine.Start);
+            var secondMiddleTrivia = root.FindTrivia(secondMiddleLine.Start);
 
-            if (position == middleLine.Start)
+            if (position == firstMiddleLine.Start)
             {
-                // we were on the ======= lines.  We only want to report here if there was no
+                // we were on the ||||||| lines.  We only want to report here if there was no
                 // conflict trivia on the <<<<<<< line (since we would have already reported the
                 // issue there.
                 if (startTrivia.RawKind == _syntaxKinds.ConflictMarkerTrivia)
                     return false;
             }
+            else if (position == secondMiddleLine.Start)
+            {
+                // we were on the ======= lines.  We only want to report here if there was no
+                // conflict trivia on the ||||||| or  <<<<<<< line (since we would have already reported the
+                // issue there.
+                if (startTrivia.RawKind == _syntaxKinds.ConflictMarkerTrivia ||
+                    (firstMiddleLine != secondMiddleLine && firstMiddleTrivia.RawKind == _syntaxKinds.ConflictMarkerTrivia))
```

Comment:
my brain is too weak to undertand this condition.  can you break into two checks?